### ISSUE TITLE
Keep getting an error with plex

### ISSUE THREAD

**Issue by HashSquatch** [Hellowlol/HTPC-Manager#495]
- [ ] Version - [a7ffcdc](https://github.com/Hellowlol/HTPC-Manager/commit/a7ffcdce698e61202fb25a5fd1d476e55ce47afc)
- [ ] Branch - Master2
- [ ] Operating system - Debian
- [ ] Python version - 2.7
- [ ] What are your (relevant) settings? local ip and port

[log](https://gist.github.com/FedoraTipper/4c63df5ffa2cb02a8e0d4a565f837857)

I need some help setting up plex with HTPC Manager. Everything else works well at the moment, but plex has an issue. 

Everything I've inputted to the plex fields is right, yet the same error keeps occurring in logs. [settings](http://i.imgur.com/xlDspzX.png)

I've also tried using my plex login, but still the same error. I do have a reverse proxy, but that shouldn't affect the local IP.

Could anyone please help me out? I'm really confused at the moment.

**Comment by Hellowlol** [comment]
See https://github.com/Hellowlol/HTPC-Manager/issues/488 
what plex version are you using?

**Comment by HashSquatch** [comment]
@Hellowlol 1.1.4.2757 

I am not sure if they had the same error as me

**Comment by Hellowlol** [comment]
Are you using ssl on plex?

**Comment by HashSquatch** [comment]
only on the reverse proxy, but nothing on local. Local is basically the default configuration. I am able to log in from my pc to the server's plex just fine, as you would do normally.

Edit: Sorry let me rephrase. Plex it's self does not have SSL on. Only the web server.

**Comment by Hellowlol** [comment]
I just updated to Version 1.1.4.2757. Works fine here. Is htpc manger and plex install on the same computer?

What happens if you do http://htpcip:port/plex/myPlexSignin?username=xxx&password=xx ? (replace xx with your username and password)

**Comment by alex-deneuvillers** [comment]
Hello, I have the same issue, and do I did what you asked. It returned me a JSON page with "Q6y8Ambxea[...]" (truncated because I'm not sure if it's confidential or not).

Plex version : 1.1.4.2757
Branch : master2
Current commit : 536904ef8f9f67177ed353ef6f822467f6c32551
Python version : 2.7.12
OS : Linux-3.14.32-xxxx-grs-ipv6-64-x86_64-with

To install it, I used docker with this image : [https://github.com/linuxserver/docker-htpcmanager](url)

**Comment by Hellowlol** [comment]
@alex-deneuvillers Thats plex autotoken so you shouldnt make that public.
- enable debug logging (settings general click save, delete the log and restart htpc manger)
- Try to open the plex module, if it fails do http://htpcip:port/plex/myPlexSignin?username=xxx&password=xx
- Open the plex module again, if it fails post the log in a gist and link it back here

**Comment by alex-deneuvillers** [comment]
Looks like it works now, thanks.

**Comment by Hellowlol** [comment]
Ok, its thats the case i think the authtoken just was invalid. Hptc manger does not renew the token unless its done manually or by restarting htpc manger. If not we would have to query plex.tv each time we want to get something from pms.

**Comment by HashSquatch** [comment]
Just an update, restarting my server this morning fixed it. ????????????? Sorry about lack of communication on my behalf
